examples/brickflow_examples/workflows/demo_wf.py

```diff
@@ -286,6 +286,7 @@ def run_snowflake_queries(*args):
             "incremental_filter": "dt='2023-10-22'",
             "dbx_data_filter": "run_dt='2023-10-21'",
             "sf_cluster_keys": "",
+            "dbx_sql":"" #Custom sql in place of dbx_catalog, database, table 
```

Comment:
this might lead to confusion . can we create one more task instead of editing this to explain this feature, May be something load_data_from _uc_to_sf_using_custom_query()

Reply:
Good idea to add the usage